apps/health_alerts/forms.py

```diff
@@ -4,6 +4,8 @@
 
 
 class HealthAlertSubscriberForm(ModelForm):
+    use_required_attribute = False
```

Comment:
I didn't know about this attribute
🆒

Reply:
yea, @vkurup and I were discussing `IntegerChoices` and `TextChoices` fields that were added in Django 3.1 that made me look for something like the above for forms in the new django version. I fortunately found the above.